Rectangle/WindowCalculation/BottomHalfCalculation.swift

```diff
@@ -8,30 +8,37 @@
 
 import Foundation
 
+private enum SupportedPositions: Int {
```

Comment:
Typically, I like using a lot of enums to clarify things and get rid of magic numbers. But in this case and any others like it, I like just straight up using the count . It's not hard to figure out what the count means without the enum, especially for anyone that uses the app.

Reply:
I like them, as I hate magic numbers, but I see your point the code is simple enough to read without them.